extensions/wikia/CommunityPage/templates/topContributors.mustache

```diff
@@ -19,7 +19,7 @@
 				<a href="{{profilePage}}">{{{avatar}}}</a>
 			</div>
 			<div class="contributor-details">
-				{{count}}. <a href="{{profilePage}}">{{userName}}</a>
+				{{count}}. <a data-tracking="top-contributors" href="{{profilePage}}">{{userName}}</a>
```

Comment:
There are already two templates for this: topContributors.mustache and topContributorsModal.mustache. The latter is used in the modal only and has the tracking label top-contributors-modal.